code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -1,6 +1,6 @@
 //Updates the mob's health from organs and mob damage variables
 /mob/living/carbon/human/updatehealth()
-	if(is_diona())
+	if(IS_DIONA())
```

Comment:
Why did you change that?
IS_DIONA is (variable) define used in chemistry

Reply:
Forgot to change this back when I was testing something. Fixed now.